modules/nf-core/paraclu/tests/main.nf.test

```diff
@@ -0,0 +1,61 @@
+nextflow_process {
+
+    name "Test Process PARACLU"
+    
+    script "../main.nf"
+    process "PARACLU"
+   
+    tag "modules"
+    tag "modules_nfcore"
+    tag "paraclu"
+
+    test("sarscov2") {
+        
+        when {
+            process {
+                """
+                input[0] = [[id:'test'],
+                    file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/bed/test.bed', checkIfExists:true)
+                    ]
+                input[1] = 1
+                """
+            }
+        }
+        then {
+            assert process.success
+            assertAll(
+                { assert snapshot(
+                    file(process.out.bed[0][1]).name,
```

Comment:
Could you add a comment to explain why you check only the name ?
`// Check only the name as it generate an empty file`